packages/database/src/modelClasses/Entity.js

```diff
@@ -363,6 +363,13 @@ export class EntityModel extends DatabaseModel {
     );
   }
 
+  /**
+   * Fetches descendant => ancestor map in given hierarchy
+   * @param {string[]} descendantCodes
+   * @param {string} hierarchyId
+   * @param {string} ancestorType
+   * @returns {Promise<Record<string, { code: string, name: string }>>} Map of descendant code to ancestor (code, name)
```

Comment:
Unfortunately I don't think that JSDocs support the `Record` type. But we can use `Object` instead:

```suggestion
   * @returns {Promise<Object<string, { code: string, name: string }>>} Map of descendant code to ancestor (code, name)
```

I believe I checked this the other day, but it would be good to confirm that this is indeed the case - using `Record` would be handy!

Reply:
Using `Record` worked fine for me 😄

Reply:
Great!!